video-player-in-flutter-using-chewie-and-getx/index.md

```diff
@@ -0,0 +1,287 @@
+
+     appBar: new AppBar( 
+
+       title: new Text("Video Player"), 
+
+     ),
+
+   );
+
+ } 
+
+}
+
+```
+
+As you see, we have a simple screen with an `appbar `, which will have the title of our player, and a `Scaffold`. The `Scaffold` will contain the background image of our video player and two rows: one with the "Play" button on top and another showing the title and URL of the selected resource.Play button will be used to execute the streaming of our video.The Title and URL are just simply for identification purposes of our video just to be sure which video is playing at that time.
+
+#### Align Widget
+This widget contains a container where we will add our components. The main purpose of the align widget is to make sure that the compononets will be placed correctly and in a more appealing location on the screen.
+
+```dart
+
+Widget build(BuildContext context) { 
+
+ return new Align( alignment: Alignment.center, //align the child at the center of the screen.
+
+ child: new Padding( 
+
+   padding: const EdgeInsets.all(16.0),//all edges will be 16 pixels from the walls.
+
+   child: new Container( 
+
+     height: 300.0, width: 300.0, 
+
+     decoration: new BoxDecoration( color: Colors.black45),
+
+   ), 
+
+ ), 
+
+ ); 
+
+}
+
+```
+
+We use the `Align` widget to align the element in the `Center` of the screen. In this case, it will be a `Container`, so we'll add a white background and a black border with inner padding of 16dp.The choice of the colour is so that the video can be seen more clearly regardless of the video quality.
+
+#### Row widget
+The Row widget will contains buttons of applications. This widget holds components from the top to bottom or vice versa.
+```dart
+
+Container build(BuildContext context) { 
+
+ return new Center( 
+
+   child: new Column( 
+
+     mainAxisAlignment: MainAxisAlignment.center, 
+
+     children: <Widget>[ new Padding( 
+
+       padding: const EdgeInsets.symmetric(vertical: 16.0), child: new Row( mainAxisAlignment: MainAxisAlignment.spaceBetween, 
+
+       children: <Widget>[ buildButton('PLAY', context), buildButton('STEP_BACKWARD', context),
+
+        ],
+
+      ), 
+
+     ), 
+
+ new Expanded(
+
+        child: new Container( 
+
+          height: 300.0, width: 300.0,
+
+          decoration: new BoxDecoration( 
+
+            borderRadius: new BorderRadius.circular(10.0)), color: Colors.black45, 
+
+            margin: const EdgeInsets.all(16.0), ), ), 
+
+ new Padding( 
+
+              padding: const EdgeInsets.symmetric(horizontal: 16.0), 
+
+              child: new Row( 
+
+                mainAxisAlignment: MainAxisAlignment.spaceBetween,
+
+                 children: <Widget>[ buildButton('PLAY', context), buildButton('STEP_FORWARD', context), 
+
+                 ], 
+
+               ), 
+
+             ),
+
+            ], 
+
+        ); 
+
+}
+
+```
+
+The `Row` contains two buttons: playing and stepping backwards in the video (to skip intros and similar stuff).
```

Comment:
Change to " The `Row` contains two buttons: playing and stepping backward in the video (to skip intros and similar stuff)."